src/components/buttons/ButtonsContainer.tsx

```diff
@@ -35,7 +36,7 @@ interface Props {
 /**
  * A consistently styled view for displaying button layouts.
  */
-export const ButtonsContainer = React.memo(({ absolute = false, fade, primary, secondary, escape, layout = 'column' }: Props) => {
+export const ButtonsContainer = React.memo(({ absolute = false, fade, primary, secondary, secondaryAsPrimary, escape, layout = 'column' }: Props) => {
```

Comment:
Seems like `secondaryAsPrimary` is an odd name. Really this is the "second" secondary button. Something like `secondary2` would be more appropriate.

Reply:
I wanted to capture that it is serving as primary in the visual heirarchy, but
skinned visually as a secondary.